docs/www/docs/cli/swa-deploy.md

```diff
@@ -157,6 +163,8 @@ Here are the options you can use with `swa deploy`:
 - `-a, --app-location <path>`: the folder containing the source code of the front-end application (default: "`.`")
 - `-i, --api-location <path>`: the folder containing the source code of the API application
 - `-O, --output-location <path>`: the folder containing the built source of the front-end application. The path is relative to `--app-location` (default: "`.`")
+- `--api-language <apiLanguage>`: the runtime language of the function (default: "`node`")
+- `--api-version <apiVersion>`: the version of the function runtime language (default: "`16`")
```

Comment:
we should not set default values. Users running a Python or .Net API might get confused.
```suggestion
- `--api-language <apiLanguage>`: the runtime language of the API
- `--api-version <apiVersion>`: the version of the API runtime language
```